doc/TROUBLESHOOT.md

```diff
@@ -101,6 +101,35 @@ Otherwise, the reference to the first non-needed object on the path (`staticX`,
 staticX -> A -> B -> disposedD
 ```
 
+Be careful not to fall into pattern to limit your fix to just releasing the reference to the disposed object. 
+Instead, you need to release just most left object that is not needed any more (first check `staticX`, then `A`, then `B`). All objects
```

Comment:
I'm not sure what a `left most object` is. Maybe the root object? Root might not be right either - what happens when there is a cycle for example?

Reply:
There are no cycles in retaining path and they are always from root. Updated the text.